x/gov/spec/01_concepts.md

```diff
@@ -56,11 +56,14 @@ arbitrary state changes.
 
 ## Deposit
 
-To prevent spam, proposals must be submitted with a deposit in the coins defined in the `MinDeposit` param. The voting period will not start until the proposal's deposit equals `MinDeposit`.
+To prevent spam, proposals must be submitted with a deposit in the coins defined in the `MinDeposit` param.
 
-When a proposal is submitted, it has to be accompanied by a deposit that must be strictly positive, but can be inferior to `MinDeposit`. The submitter doesn't need to pay for the entire deposit on their own. If a proposal's deposit is inferior to `MinDeposit`, other token holders can increase the proposal's deposit by sending a `Deposit` transaction. The deposit is kept in an escrow in the governance `ModuleAccount` until the proposal is finalized (passed or rejected).
+When a proposal is submitted, it has to be accompanied by a deposit that must be strictly positive, but can be inferior to `MinDeposit`. The submitter doesn't need to pay for the entire deposit on their own.
+The newly created proposal is stored in an _inactive proposal queue_ and stays there until its deposit passes the `MinDeposit`. Other token holders can increase the proposal's deposit by sending a `Deposit` transaction.
+If a proposal won't pass the `MinDeposit` before the deposit end time (the time when users can make deposits), it will be killed: removed from the state and the deposit will be burned (x/gov EndBlocker).
+When a proposal has at least 2 deposits (so at least one more account must call `Depoist`) and passes the `MinDeposit` threshold before the deposit end time, it will be moved into the _active proposal queue_ and enters voting period.
```

Comment:
You don't technically need to have two deposits, right? The proposal proposer/submitter can provide an initial deposit that >= `MinDeposit`. I think we should make this clear here.

Reply:
I was asking myself the same question when I was writing it and verified it. You need at least two deposit (one from the MsgSubmitProposal, and one more).

Reply:
What if the deposit from `MsgSubmitProposal` is >= `MinDeposit`? Can you point me to the code?

Reply:
In [SubmitProposal](https://github.com/cosmos/cosmos-sdk/blob/robert%2Fcosmvisor-file-watch/x/gov/keeper/proposal.go#L41) we always insert the new proposal into the inactive queue.

Reply:
Ohh goodness, that's not a good UX. `MsgSubmitProposal` handler should handle this, but that's another story. Ok. this is fine for now then.

Reply:
@robert-zaremba I think two deposits are not necessary, afaik one deposit which is >= `MinDeposit` is enough to make the proposal voting period start. 
Ref: https://github.com/cosmos/cosmos-sdk/blob/master/x/gov/keeper/msg_server.go#L36-L39
https://github.com/cosmos/cosmos-sdk/blob/master/x/gov/keeper/msg_server.go#L50-L54

Reply:
Two deposits are not necessary as far as I can tell. This can also be tested by including a deposit when submitting a proposal using the CLI, which immediately starts the voting period and does not require a second deposit.

Reply:
You're absolutely right. Thanks for catching this. @robert-zaremba please update the doc accordingly 👍

Reply:
Ah, all right. The `MsgSubmitProposal` adds it to the inactive queue, but later in the `AddDepoist` call we are moving it to the active queue: https://github.com/cosmos/cosmos-sdk/blob/d4d25f5e18e77e3040df49f0f37006fb50ffd501/x/gov/keeper/deposit.go#L139

Updating the spec.